src/renderer/translations/tokens/en-US.json

```diff
@@ -0,0 +1,213 @@
+  "verification-body": "Verification Body",
+  "verification-report-date": "Verification Report Date",
+  "label": "Label",
+  "label-type": "Label Type",
+  "label-link": "Label Link",
+  "validity-period-start-date": "Validity Period Start Date",
+  "validity-period-end-date": "Validity Period End Date",
+  "crediting-period-start-date": "Crediting Period Start Date",
+  "crediting-period-end-date": "Crediting Period End Date",
+  "unit-quantity": "Unit Quantity",
+  "country": "Country",
+  "in-country-region": "In Country Reason",
+  "geographic-identifier": "Geographic Identifier",
+  "file-id": "File ID",
+  "rating-type": "Rating Type",
+  "rating-range-lowest": "Rating Range Lowest",
+  "rating-range-highest": "Rating Range Highest",
+  "rating": "Rating",
+  "rating-link": "Rating Link",
+  "relationship-type": "Relationship Type",
+  "unit-block-start": "Unit Block Start",
+  "unit-block-end": "Unit Block End",
+  "in-country-jurisdiction-of-owner": "In-Country Jurisdiction of Owner",
+  "project-location-id": "Project Location ID",
+  "unit-status-reason": "Unit Status Reason",
+  "unit-registry-link": "Unit Registry Link",
+  "marketplace-identifier": "Marketplace Identifier",
+  "marketplace-link": "Marketplace Link",
+  "select-issuance": "Select Issuance",
+  "select-organization": "Select Organization",
+  "transfer": "Transfer",
+  "create-project-transfer": "Create Project Transfer",
+  "transfer-project": "Transfer Project",
+  "no-pending-transfers": "No pending transfers",
+  "download-offer-file": "Download Offer File",
+  "cancel-offer": "Cancel Offer",
+  "failed-to-cancel-offer": "Failed to cancel offer",
+  "confirm-cancel-offer": "Confirm Cancel Offer",
+  "do-not-cancel-offer": "Do Not Cancel Offer",
+  "cancelling-this-offer-will-invalidate-the-corresponding-offer-file": "Cancelling this offer will invalidate the corresponding offer file",
+  "import-transfer-file": "Import Transfer File",
+  "reject-transfer-offer": "Reject Transfer Offer",
+  "accepting-this-offer-will-write-the-associated-changes-transferring-ownership-of-the-project": "Accepting this offer will write the associated changes transferring ownership of the project",
+  "the-transfer-cannot-be-revoked-once-committed": "The transfer cannot be revoked once committed",
+  "accept-and-commit-transfer": "Accept and Commit Transfer",
+  "failed-to-accept-offer": "Failed to accept offer",
+  "confirm-project-transfer": "Confirm Project Transfer",
+  "the-project-has-been-successfully-transferred-and-the-associated-changes-have-been-committed": "The project has been successfully transferred and the associated changes have been committed",
+  "project-transfer-error": "Project Transfer Error",
+  "failed-to-commit-project-transfer": "Failed to commit project transfer",
+  "no-changes-have-been-made": "No changes have been made",
+  "api-host-loaded-from-configuration": "API host loaded from configuration",
+
+  "projects-view": "Projects View",
```

Comment:
i know that the example tokens in this file have capitalization in the token itself, but please use no capitalization in any tokens you add and instead wrap the `<FormattedMessage>` tags in `<p>` tags or the parent div and use tailwind classes like `capitalize` or the `sentence-case` class i had you add